reference/features/filtering_and_faceted_search.md

```diff
@@ -0,0 +1,318 @@
+
+::: warning
+`filter` is a new search parameter introduced in MeiliSearch v0.21. It combines functionality previously present in v0.20's `filters` and `facetFilters`.
+
+Both `filters` and `facetFilters` have been removed from v0.21.
+:::
+
+### Conditions
+
+Conditions are a filter's basic building blocks. They are always written in the `attribute OPERATOR value` format, where:
+
+- `attribute` is the attribute of the field you want to filter on
+- `OPERATOR` is the comparison operator and can be `=`, `!=`, `>`, `>=`, `<`, or `<=`
+- `value` is the value condition for the filter
+
+::: note
+`>`, `>=`, `<`, and `<=` only operate on numeric values and will ignore all other types of values.
+
+When operating on strings, `=` and `!=` are **case-insensitive**.
+:::
+
+#### Examples
+
+A basic condition could request movies containing the `horror` genre:
+
+```
+genres = horror
+```
+
+Note that string values containing whitespace must be enclosed in single or double quotes:
+
+```
+director = 'Jordan Peele'
+director = "Tim Burton"
+```
+
+Another condition could request movies released after 18 March 1995 (written as 795484800 in UNIX Epoch time):
+
+```
+release_date > 795484800
+```
+
+::: warning
+As no specific schema is enforced at indexing, the filtering engine will try to coerce the type of `value`. This can lead to undefined behaviour when big floats are coerced into integers and reciprocally. For this reason, it is best to have homogeneous typing across fields, especially if numbers tend to become large.
+:::
+
+### Filter expressions
+
+You can build filter expressions by grouping basic conditions. Filter expressions can be written as strings, arrays, or a mix of both.
+
+#### Creating filter expressions with strings
+
+String expressions combine conditions using three logical connectives—`AND`, `OR` and `NOT`— and parentheses:
+
+- `NOT` only returns documents that do not satisfy a condition : `NOT genres = horror`
+- `AND` operates by connecting two conditions and only returns documents that satisfy both of them: `genres = horror AND director = 'Jordan Peele'`
+- `OR` connects two conditions and returns results that satisfy at least one of them: `genres = horror OR genres = comedy`
+
+::: tip
+String expressions are read left to right. `NOT` takes precedence over `AND` and `AND` takes precedence over `OR`. You can use parentheses to ensure expressions are correctly parsed.
+
+For instance, if you want your results to only include `comedy` and `horror` movies released after March 1995, the parentheses in the following query are mandatory:
+
+`(genre = horror OR genre = comedy) AND release_date > 795484800`
+
+Failing to add these parentheses will cause the same query to be parsed as:
+
+`genre = horror OR (genre = comedy AND release_date > 795484800)`
+
+Translated into English, the above expression will only return comedies released after March 1995 or horror movies regardless of their `release_date`.
+:::
+
```

Comment:
You can add that when using `GET /search` only the expression with strings works.
So not arrays and no mixed